contrib/snapshot/bitcoin.conf

```diff
@@ -0,0 +1,5 @@
+regtest=1
+disablewallet=0
+printtoconsole=1
+rpcuser=mysecretrpcuser
+rpcpassword=mysecretrpcpassword
```

Comment:
```suggestion
rpcuser=user
rpcpassword=password
```

Reply:
just to make it simpler

Reply:
Thanks! I copied this from the README of bitcoin docker 👍🏼